packages/verkle/src/node/leafNode.ts

```diff
@@ -1,15 +1,21 @@
 import { equalsBytes, intToBytes, setLengthLeft, setLengthRight } from '@ethereumjs/util'
 
 import { BaseVerkleNode } from './baseVerkleNode.js'
-import { NODE_WIDTH, VerkleNodeType } from './types.js'
+import {
+  DEFAULT_LEAF_VALUES,
+  DELETED_LEAF_VALUE,
+  NODE_WIDTH,
+  VerkleNodeType,
+  createDeletedLeafValue,
+} from './types.js'
 import { createCValues } from './util.js'
 
 import type { VerkleNodeOptions } from './types.js'
 import type { VerkleCrypto } from '@ethereumjs/util'
 
 export class LeafNode extends BaseVerkleNode<VerkleNodeType.Leaf> {
   public stem: Uint8Array
-  public values: Uint8Array[] // Array of 256 possible values represented as 32 byte Uint8Arrays
+  public values: (Uint8Array | 0 | 1)[] // Array of 256 possible values represented as 32 byte Uint8Arrays or 0 if untouched or 1 if deleted
```

Comment:
Could we, instead of numbers, use an enum? I think that would be clearer in usage

Reply:
I've gone ahead and done that, let me know what you think

Reply:
These edits look great! Thanks!